kfs-core/src/main/resources/institutional-config.properties

```diff
@@ -1,3 +1,6 @@
+batchinvoker.directory.towatch=${work.directory}/control
+batchinvoker.directory.history=${work.directory}/control/history
```

Comment:
Yup, these are pulled by Spring in spring-sys.xml:83,86, in order to inject instance variable values into RemoteBatchJobInvoker.java:44,45. The instance variables are then used to build directory paths, where various files are watched/shuffled.

More specifically, the 'control' directory has several purposes:
1. Holds a 'runnable' file, which signals to the Batch Invoker to be active for that specific host, encoded as the hostname into the filename
2. Acts as a drop box where Rhubarb/scripts/humans drop a 'run' file, which informs the Batch Invoker which job to schedule, also encoded into the filename
3. Holds the 'status' file, where the Batch Invoker updates a timestamp that humans can inspect, and when done, either a "success" or "failure" message, along w/ any error messages that resulted in a failure

The 'history' directory, is where the Batch Invoker moves a completed status file. This is also where Rhubarb looks to find it's status for previously issued job thus signifying completion, and the contents of which determines what exit status Rhubarb issues. Currently the consumers of this exit status are Control-M, and the new application Rhubarb Farmer.